GitUI/CommandsDialogs/SettingsDialog/Pages/ChecklistSettingsPage.cs

```diff
@@ -713,5 +749,10 @@ private static void RenderSettingNotRecommended(Button settingButton, Button set
             settingButton.Text = text;
             settingFixButton.Visible = true;
         }
+
+        private void GcmDetectedFix_Click(object sender, EventArgs e)
+        {
+            Process.Start("https://github.com/gitextensions/gitextensions/issues/3511#issuecomment-313633897");
```

Comment:
💭 If we did have a way of hosting web pages in GE (such as CefSharp) then we could open web pages in tabs, with an option to launch externally.

💭 It should be possible to instrument in-process GitHub/BitBucket/etc pages with custom JavaScript to add links back into GitExtensions and its plugins. That level of integration could be nice. Instrumenting pages is prone to breakages and requires the ability to patch rapidly. We'd need to check their terms of service, but I think there is prior art by way of browser extensions.

I've used CefSharp a little before. It is not a dependency to be taken lightly. It's large on disk, heavy on RAM. But then so are external browsers. We should run tests to compare, and if they're equivalent then it's not a barrier.

CefSharp's release schedule works with a bounty system where people donate and when thresholds are met then the work is completed. The core guy Alex (who's somewhere in Australia I believe) has expressed on Gitter in the past that he needed to step away from the project for a while. Since then he's come back and put out a big release and appears to be active. I think we should only consider hosting browsers as a feature if we have backup options from other active libraries.

Reply:
WRT: the link - I did ponder upon this for a bit. Another option (and perhaps a better one) is to link to our Wiki page, which should be a more permanent placement than a comment in a thread. The Wiki page then provide necessary explanations and links.

WRT: CefSharp - we use it at work, and I know a little bit about it. I've also looked at it as part of #4800. Indeed it is a significant dependency and, at this stage, I am not totally convinced it is absolutely necessary.

Reply:
Linking to Wiki is done for Git, I suggest the same is done here.

Reply:
Wiki page added:
https://github.com/gitextensions/gitextensions/wiki/Obsolete-git-credential-winstore.exe

Reply:
Awesome thanks.
I renamed the page to https://github.com/gitextensions/gitextensions/wiki/How-To:-fix-GitCredentialWinStore-missing though.

Reply:
Happy to help where and when I can.
Pleased to see how how simple it was in the end to get Markdown to show the settings indented correctly. I was having trouble in the Wiki page as the result was different to how it appeared in comments.